### PR TITLE
Improve issue template

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Replaces 3 possible states (absent, present unchecked, present checked) with 2.

## Readiness checklist

* [ ] I added unit tests for new functionality or bugfixes.
* [ ] I added integration tests for new functionality or bugfixes.
* [ ] I added compatibility tests for new functionality or bugfixes.
* [ ] I made spot refactoring.
* [ ] I updated the documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
 name: Bug
-description: Some user-visible feature works incorrectly
+description: Some feature works incorrectly
```
Comment (AlekSi) [inline .github/ISSUE_TEMPLATE/bug.yml:4]: That's still implied, but now it is less weird to users: we should not convince users that their problems are user-visible, and, hopefully, we would not forget that

File: .github/ISSUE_TEMPLATE/chore.yml [hunk 2]
```diff
@@ -14,23 +14,26 @@ body:
...
-        - [ ] compatibility tests added
-        - [ ] spot refactoring done
-        - [ ] documentation updated
-        - [ ] anything else?
+        - unit tests added;
+        - integration tests added;
+        - compatibility tests added;
+        - spot refactorings done;
+        - user documentation updated;
+        - something else?
```
Comment (rumyantseva) [inline .github/ISSUE_TEMPLATE/chore.yml:37]: As far as I understand English grammar, if bullet points are full sentences, they should start from capital letters - https://www.linkedin.com/pulse/how-capitalize-punctuate-bullet-points-l-michelle-baker-phd/.

```suggestion
        - Unit tests added.
        - Integration tests added.
        - Compatibility tests added.
        - Spot refactorings were done.
        - User documentation updated.
        - Something else?
```

**Review by rumyantseva — APPROVED** [review]
LGTM.

I have one tiny suggestion, but I'm not sure about it, so I leave it up to you.

File: .github/ISSUE_TEMPLATE/chore.yml (hunk 2 above)
Comment (AlekSi) [reply .github/ISSUE_TEMPLATE/chore.yml:37]: Isn't that example B from that article rather than example D?

Comment (rumyantseva) [reply .github/ISSUE_TEMPLATE/chore.yml:37]: I'm not sure :) "Unit tests added" sounds like a complete sentence. But I'm ok with your proposal too.

**Review by w84thesun — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Pull request merged**